server/item/register.go

```diff
@@ -23,6 +23,7 @@ func init() {
 	world.RegisterItem(TurtleShell{})
 
 	world.RegisterItem(Bucket{})
+	world.RegisterItem(MilkBucket{})
```

Comment:
One thing I'm unsure about is the fact that `Bucket` and `MilkBucket` are split up, however I do understand that there's probably not a great way around this. Maybe you could rename `Bucket` to `LiquidBucket`?

Reply:
sure